AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -127,6 +138,16 @@ @interface MSCrashes ()
  */
 @property BOOL didCrashInLastSession;
 
+/**
+ * Indicates if the app did receive a low memory warning in the last session.
+ * It may happen that low memory warning where send but couldn't be logged, since iOS killed the app before updating the flag in the
+ * filesystem did complete. This property may be true in case of low memory kills, but it doesn't have to be! Apps can also be killed
+ * without the app ever receiving a low memory warning. Also the app could have received a low memory warning, but the reason for being
+ * killed was actually different.
```

Comment:
please replace this with some grammatical changes/clarifications

```
Indicates that the app received a low memory warning in the last session.
It is possible that a low memory warning was sent but couldn't be logged if iOS killed the app before updating the flag in the filesystem.
Apps can also be killed without receiving a low memory warning, or receive the warning, but crash for another reason.
```

Reply:
@michaelsmitha ok.
(the current version is from HockeyApp 😃)

https://github.com/bitstadium/HockeySDK-https://github.com/bitstadium/HockeySDK-iOS/blob/dabf7f9f66b27a4dca089ad6dbbece04bae7d7dd/Classes/BITCrashManager.h#L369-L387

Reply:
@MatkovIvan haha gotcha, it was really hard to read :)